src/components/utils/ControlledAlert/ControlledAlert.tsx

```diff
@@ -16,8 +16,11 @@ export interface ControlledAlertProps {
   /** When the alert is discardable, this is the callback that gets fired on discard */
   onClose: () => void;
 
+  /** The alert severity */
```

Comment:
erm it's not "severity" per say.

Reply:
Do you think "severity" better describes the prop? I was in between "severity" and "type".